Example/CotterIOS/AppDelegate.swift

```diff
@@ -8,23 +8,30 @@
 
 import UIKit
 import Cotter
+import UserNotifications
 
 @UIApplicationMain
 class AppDelegate: UIResponder, UIApplicationDelegate {
 
     var window: UIWindow?
 
-
-    func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
+    func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey : Any]? = nil) -> Bool {
         // Override point for customization after application launch.
         
-        guard let apiKeyID = Environment.shared.COTTER_API_KEY_ID else {
-            print("Please set COTTER_API_KEY_ID in your XCode environment variables!")
-            return false
+        // set credentials
+        var apiKeyID: String
```

Comment:
Can we just do `var apiKeyID = "fb1f9830-574c-4b0d-bafb-68713ed927a2"` and `var apiSecreyKey = "KLKqAQ6QXEScmQbYvtJm"` here instead of else below?